src/cli/commands/bitswap/stat.js

```diff
@@ -16,22 +16,22 @@ module.exports = {
         throw err
       }
 
-      stats.Wantlist = stats.Wantlist || []
-      stats.Wantlist = stats.Wantlist.map((entry) => {
+      stats.wantlist = stats.wantlist || []
+      stats.wantlist = stats.wantlist.map((entry) => {
         const buf = Buffer.from(entry.cid.hash.data)
         const cid = new CID(entry.cid.version, entry.cid.codec, buf)
         return cid.toBaseEncodedString()
       })
-      stats.Peers = stats.Peers || []
+      stats.peers = stats.peers || []
 
       print(`bitswap status
-  blocks received: ${stats.BlocksReceived}
-  dup blocks received: ${stats.DupBlksReceived}
-  dup data received: ${stats.DupDataReceived}B
-  wantlist [${stats.Wantlist.length} keys]
-    ${stats.Wantlist.join('\n    ')}
-  partners [${stats.Peers.length}]
-    ${stats.Peers.join('\n    ')}`)
+  blocks received: ${stats.blocksReceived}
+  dup blocks received: ${stats.dupBlksReceived}
+  dup data received: ${stats.dupDataReceived}B
+  wantlist [${stats.wantlist.length} keys]
+    ${stats.wantlist.join('\n    ')}
+  partners [${stats.peers.length}]
+    ${stats.peers.join('\n    ')}`)
     })
   }
 }
```

Comment:
After Rebase, enable the CLI Bitswap test `stat`

Reply:
Done and it works!!